swift/ql/lib/codeql/swift/elements/decl/Decl.qll

```diff
@@ -1,4 +1,5 @@
-// generated by codegen/codegen.py, remove this comment if you wish to edit this file
 private import codeql.swift.generated.decl.Decl
 
-class Decl extends Generated::Decl { }
+class Decl extends Generated::Decl {
+  override string toString() { result = super.toString() }
```

Comment:
Actually, no, see my dismissal comment.

Reply:
Though I guess I could just undo the toString change on DeclRefExp... Since the lack of a toString on the underlying Decl seems to indicate a missing Builtin.

Reply:
Ah, I see. In that case I'm happy with this `toString`. I agree that `Decl`s _should_ all have `toString`s, so I'm also happy with undoing the `toString` change if you prefer.